src/renderers/webgl/WebGLMaterials.js

```diff
@@ -17,7 +17,7 @@ function WebGLMaterials( renderer, properties ) {
 
 	function refreshFogUniforms( uniforms, fog ) {
 
-		fog.color.getRGB( uniforms.fogColor.value, getUnlitUniformColorSpace( renderer ) );
+		fog.color.getRGB( uniforms.fogColor.value, ColorManagement.workingColorSpace );
```

Comment:
@donmccurdy

Does this make sense or should it just be this?

```js
fog.color.getRGB( uniforms.fogColor.value, LinearSRGBColorSpace );
```

Reply:
Working color space! 👍🏻 

Leaving the parameter out would also be fine — our [docs](https://threejs.org/docs/?q=color#api/en/math/Color.getRGB) and [code](https://github.com/mrdoob/three.js/blob/ce0b3d0415c4bc87a8b4ad01c5d21cc26c6ec5eb/src/math/Color.js#L412) differ on this, but the code does the right thing.

Reply:
I'll remove the parameter then 👌